app/src/main/res/layout/market_picker_bottom_sheet.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
How does this sheet behave when it has a large number of items, enough to be scrolled?

Reply:
Yes it behaves as we want, bottomsheet extends and if it is still to many markets you can scroll the list